Production_scripts/Process_extracts/Produce_extract_tests/extract_homelessness_tests.R

```diff
@@ -0,0 +1,43 @@
+####################################################
+# Name of file - extract_homelessness_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               homelessness processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Homelessness", ext = "rds")) %>%
+  mutate(smrtype = stringr::str_trim(smrtype, "both"))
+
+# Read current SLF episode file and filter for homelessness records (recid HL1)
+existing_data <- read_slf_episode(year, recid = "HL1") %>%
+  dplyr::rename(chi = anon_chi) %>%
+  dplyr::select(colnames(new_data))
+
+# Produce comparison-------------------------------------
+# Compare new file with existing slf data
+
+comparison <- produce_slf_homelessness_tests(
+  new_data = new_data,
+  existing_data = existing_data
+)
```

Comment:
```suggestion
comparison <- produce_test_comparison(
  produce_source_extract_tests(existing_data),
  produce_source_extract_tests(new_data)
)
Can wrap the old and new dataframes in the `produce_test_comparison` function. This example is from Acute tests. Will need tweaking to reflect homelessness variables and use updated function with `sum_mean_vars` parameters
```

Reply:
Done, other things changed as mentioned with the `produce_test_comparison` function